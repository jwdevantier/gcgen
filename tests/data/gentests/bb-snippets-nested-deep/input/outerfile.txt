
My outer file...

First foo:
# <<? foo ?>>
# <<? /foo ?>>

Then bar:
# <<? bar ?>>
# <<? /bar ?>>