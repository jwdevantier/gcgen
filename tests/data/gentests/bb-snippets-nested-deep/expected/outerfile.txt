
My outer file...

First foo:
# [[start foo
foo from outer ctx!
scope["some-var"]: <<some-outer-val>>
/outer foo end
# end]]

Then bar:
# [[start bar
bar from outer ctx!
# end]]